README.md

```diff
@@ -170,11 +181,9 @@ Anyone who has been active in the foundation or one of its member projects, as d
 - Even Stensberg ([@evenstensberg](https://github.com/evenstensberg))
 - Jean Burellier ([@sheplu](https://github.com/sheplu))
 - Jon Church ([@jonchurch](https://github.com/jonchurch))
-- Jordan Harband ([@ljharb](https://github.com/ljharb))
 - Jory Burson ([@jorydotcom](https://github.com/jorydotcom))
 - Marco Ippolito ([@marco-ippolito](https://github.com/marco-ippolito))
 - Mert Can Altin ([@mertcanaltin](https://github.com/mertcanaltin))
-- Michael Dawson ([@mhdawson](https://github.com/mhdawson))
```

Comment:
this change isn't correct; the two memberships are distinct, and remain even if the voting members lose the election to be one.

Reply:
oh?  some people who are voting members were not listed as regular members, so that's the thing that needs correcting then it sounds like

Reply:
Technically I think an impact project can appoint a voting member who's not a regular member, so that might not be wrong either.

Reply:
but there's language like this in governance:

> Note: Former Voting members whose terms have just ended will automatically become Regular members, unless they indicate otherwise.

Reply:
so it sounds like technically you do not dual-card

Reply:
ah ok then yes, every voting member should also be in the regular member list :-)

Reply:
while it doesn't seem to make any substantive difference, I guess it's easier to leave them in than to take them out and then have to re-add them later when folks may no longer be voting members so...  you got it, dude

Reply:
@ljharb no rush, but this PR is still blocked on your change request